pytube/extract.py

```diff
@@ -396,3 +398,48 @@ def apply_descrambler(stream_data: Dict, key: str) -> None:
         ]
 
     logger.debug("applying descrambler")
+
+
+def initial_data(watch_html: str) -> str:
+    """
+    Extracts the initial_data json from the watch_html page.
+    This is used for extraction of copyright notices located at the bottom
+    of the video description
+    @param watch_html: Html of the watch page
+    @return:
+    """
+    initial_data_pattern = r"window\[['\"]ytInitialData['\"]]\s*=\s*([^\n]+)"
+    try:
+        match = regex_search(initial_data_pattern, watch_html, 1)
+    except RegexMatchError:
+        return "{}"
+    else:
+        return match[:-1]
+
+
+def metadata(initial_data) -> Optional[YouTubeMetadata]:
+    """Get the metadata for the video.
```

Comment:
adding example of some metadata would be helpful here when looking back at code :)